sourcegraph/sourcegraph
# Code Insights: Update insight template examples

Closes https://github.com/sourcegraph/sourcegraph/issues/31355

## Test plan 

- Check that all examples are presented on the about code insights page in the template section 
- Check that all templates have the right values from the Approved/Tested queries spreadsheet
